src/ast/nodes/ExportDefaultDeclaration.ts

```diff
@@ -146,7 +146,9 @@ export default class ExportDefaultDeclaration extends NodeBase {
 			code.overwrite(
 				this.start,
 				declarationStart,
-				`${cnst} ${this.variable.getName(getPropertyAccess)} = exports('${systemExportNames[0]}', `
+				`${cnst} ${this.variable.getName(getPropertyAccess)} = exports(${JSON.stringify(
+					systemExportNames[0]
+				)}, `
```

Comment:
I included this change in master now as it is not central to this PR and makes the diff much smaller (also, I do not think it is strictly necessary except to have some symmetry in what quotes are used for exports=.